src/stores/GuildMemberRoleStore.js

```diff
@@ -138,6 +149,24 @@ class GuildMemberRoleStore extends Collection {
    *   .catch(console.error);
    */
   set(roles, reason) {
+    if (roles === undefined) return Promise.resolve(this.member);
```

Comment:
This could be as suggested to also handle null instaces
```suggestion
	// eslint-disable-next-line eqeqeq
    if (roles == undefined) return Promise.resolve(this.member);
```